src/my-options.html

```diff
@@ -34,6 +37,7 @@
       <p>Vision: </p>
       <p>HRA/HSA: </p>
       <p>HRA/HSA Company Contribution: </p>
+      <div class="horizontal start-justified layout"><a href="/questions"><paper-button raised>Back</paper-button></a></div>
```

Comment:
Following the example set here:

To use paper-button as a link, wrap it in an anchor tag. Since paper-button will already receive focus, you may want to prevent the anchor tag from receiving focus as well by setting its tabindex to -1.
```
<a href="https://www.polymer-project.org/" tabindex="-1">
  <paper-button raised>Polymer Project</paper-button>
</a>
```
https://elements.polymer-project.org/elements/paper-button